core/predefined_mci.js

```diff
@@ -124,6 +124,12 @@ const PREDEFINED_MCI_GENERATORS = {
     UN: function userName(client) {
         return client.user.username;
     },
+    UZ: function sanitizedUserName(client) {
```

Comment:
Nice! We'll need to get these in the MCI docs

Reply:
Yeah I actually probably need to do a few more of these too and think about the split - I.e I could also add first name / last name ones etc among others. Probably needs some more thinking on what is appropriate here